chain/rpc/src/rpc.rs

```diff
@@ -39,6 +49,10 @@ pub struct RpcOperationsConfig {
     ///
     /// Defaults to null address.
     pub module_address: Address,
+    /// Address of the node's safe contract.
+    ///
+    /// Defaults to null address.
```

Comment:
These addresses should not default to `0x00...` addresses should be valid at this point.

Reply:
`0x00...0` is a valid default on-chain address though.